cmd/fastly/main.go

```diff
@@ -39,11 +39,103 @@ func main() {
 		out            io.Writer = common.NewSyncWriter(os.Stdout)
 	)
 
+	// Extract a subset of configuration options from the local application directory.
+	var file config.ConfigFile
+	err := file.Read(config.FilePath)
+	if err != nil {
+		text.Output(out, `
+			We were unable to locate a local configuration file required to use the CLI.
```

Comment:
As noted in my design doc... 

**Should we be informing users of the existence of this file?** 
Currently I am displaying messages to the user to inform them when the local configuration file is missing and whether it’s stale, and whether we’ve been able to update it successfully. This feels overly noisy and not necessarily of value to a user. It might be better to hide this output behind the `-v`, `-verbose` flag (or `-d`, `-debug` †)

> † as noted by https://clig.dev/#arguments-and-flags `-v` can be confused with `--version`.